promises.cf

```diff
@@ -57,6 +57,10 @@ body common control
 
       version => "CFEngine Promises.cf 3.7.0";
 
+      cfengine_3_7.(debian||redhat)::
+          package_inventory => { $(package_module_knowledge.platform_default) };
+          package_module => $(package_module_knowledge.platform_default);
```

Comment:
The only way in 3.7 that I can think to support setting them is having promises.cf and promsie_3.7.cf where the correct version is chosen in cf-execd.

In 3.9 we could use the IF cf_version thing. But that's not available in 3.6 or 3.5 (we try to maintain 2 version back compatibility).

I believe we had a similar situation with protocol_version in 3.6. You could not enable the setting until all your hosts were on 3.6 because that control option was not valid in 3.5. So probably a commented out block is the best and least complex thing to do.